tests/integration/test_epicor.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+from viadot.sources import Epicor
+from viadot.config import local_config
+from viadot.exceptions import DataRangeError
+
+
+@pytest.fixture(scope="session")
+def epicor():
+    epicor = Epicor(
+        base_url=local_config.get("Epicor").get("test_url"),
+        config_key="Epicor",
+        filters_xml="""
+    <OrderQuery>
+        <QueryFields>
+            <CompanyNumber>001</CompanyNumber>
+            <BegInvoiceDate>2022-05-16</BegInvoiceDate>
+            <EndInvoiceDate>2022-05-16</EndInvoiceDate>
+            <RecordCount>3</RecordCount>
+        </QueryFields>
+    </OrderQuery>""",
+    )
+    yield epicor
+
+
+@pytest.fixture(scope="session")
+def epicor_error():
+    epicor_error = Epicor(
+        base_url=local_config.get("Epicor").get("test_url"),
+        config_key="Epicor",
```

Comment:
I would use all caps as for all other cred keys (except DuckDB, not sure what happened there :D)